### ISSUE TITLE
Dashboard A11y: Scroll to Top Button - cannot be focused

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#4201]
## Bug Description

Currently cannot tab to and focus the scroll to top button in list view that shows up on views that require scroll. 

## Expected Behaviour

Be able to use the keyboard to focus on the scroll to top button and trigger button with keyboard. 

## Steps to Reproduce

Try to tab to while in my stories list view and focus on the scroll to top button, see that you cannot. 

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

- Fix should include karma tests

**Comment by BrittanyIRL** [comment]
So this actually can be reached, but the grid content that comes before it needs to be fixed to use a keyboard more effectively instead of tabbing through card levels. So once that's handled this will resolve. Leaving it open to double check.

**Comment by swissspidy** [comment]
Can this button even be reached due to the infinite scrolling? 🤔  Hard to tab to that button when new stories are constantly loaded

**Comment by BrittanyIRL** [comment]
@swissspidy it will be a lot easier once we get the grid keyboard navigation into this page (#4200 for the list view) and i'll make a ticket for the dashboard grids to have the same (https://github.com/google/web-stories-wp/issues/4296). Didn't realize that wasn't a ticket yet. I'm keeping this ticket around to help track that update.

**Comment by carloskelly13** [comment]
PR #4317 has the commits for this ticket.

**Comment by csossi** [comment]
Verified in QA
